apps/Standalone/src/designer/app/AzureLogicAppsDesigner/laDesigner.tsx

```diff
@@ -475,6 +475,91 @@ const getDesignerServices = (
           true /* supportsAuthenticationParameter */
         );
       },
+      getWorkflowSubSchema: (args: any) => {
```

Comment:
NIT: I know that the rest of the callbacks have any as type, but can we specify this one?

Reply:
this is a random function which i added for testing purpose, it is not a known one hence I would not want to add it. Also I am going to use that for end to end testing.